app/panel/utils/utils.js

```diff
@@ -234,3 +234,120 @@ export function setTheme(doc, themeName, theme) {
 		document.head.appendChild(themeStyle);
 	}
 }
+/**
```

Comment:
@zarembsky let's remove this dynamic resizing code and revert back to what we had previously.  I want to hold the translation providers accountable for delivering language strings that fit our character count, rather than trying to adapt our layouts to fit what they throw at us.